go.mod

```diff
@@ -37,7 +37,6 @@ require (
 	github.com/onsi/gomega v1.17.0
 	github.com/ory/go-acc v0.2.6
 	github.com/pkg/errors v0.9.1
-	github.com/sclevine/agouti v0.0.0-20190613051229-00c1187c74ad
```

Comment:
:thinking:

Reply:
is there something I should do about this?

Reply:
I believe it's because #1555 removed the use of that module, and I forgot to put `go mod tidy` as part of the static checks. I've added it in #1655.